vpr/src/base/netlist.tpp

```diff
@@ -269,9 +269,18 @@ PortType Netlist<BlockId, PortId, PinId, NetId>::port_type(const PortId port_id)
 template<typename BlockId, typename PortId, typename PinId, typename NetId>
 std::string Netlist<BlockId, PortId, PinId, NetId>::pin_name(const PinId pin_id) const {
     BlockId blk = pin_block(pin_id);
-    PortId port = pin_port(pin_id);
 
-    return block_name(blk) + "." + port_name(port) + "[" + std::to_string(pin_port_bit(pin_id)) + "]";
+    return block_name(blk) + "." + port_bit_name(pin_id);
+}
+
+template<typename BlockId, typename PortId, typename PinId, typename NetId>
+std::string Netlist<BlockId, PortId, PinId, NetId>::port_bit_name(const PinId pin_id) const {
```

Comment:
The naming convention we typically follow is for the first part of the member function to by the 'type' of component the function operates on. In this case it takes a PinId, so it would be named something like `pin_port_bit_name()`.